ExampleFhirClient/AccessTokenHelper.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens;
+using System.Net.Http;
+using System.Security.Claims;
+using System.Security.Cryptography.X509Certificates;
+
+namespace ExampleFhirClient
+{
+	static class AccessTokenHelper
+	{
+		public static string GetAccessToken( Uri baseAddress )
+		{
+			var cert = GetClientCertificate( "CN=jwt.careevolution.com" );
+			var clientId = "JWTClientCredentials";
+			var tokenvalue = CreateClientCredentialsJWT( cert, clientId, baseAddress );
+
+			using ( var client = new HttpClient { BaseAddress = baseAddress } )
```

Comment:
Creating HttpClient each time is bad (...I did not know...) so maybe use a static variable for it - just not to spread bad examples..

Reply:
I've updated code to not create HttpClient each time.